docs/diagrams/ModelClassDiagram.puml

```diff
@@ -40,6 +41,7 @@ UniquePersonList --> "~* all" Person
 Person *--> Name
 Person *--> Phone
 Person *--> Email
+Person *--> IdentityNumber
```

Comment:
Might have to play with the ordering to make it flow nicely but it is on the low priority side

Reply:
Trivial Comment